dali/python/nvidia/dali/ops.py

```diff
@@ -264,22 +265,25 @@ def _load_ops():
     _cpu_gpu_ops = _cpu_ops.union(_gpu_ops).union(_mixed_ops)
     _support_ops = _support_ops.union(set(b.RegisteredSupportOps()))
     for op_name in _cpu_gpu_ops:
-        setattr(sys.modules[__name__], op_name,
-                python_op_factory(op_name, op_device = "cpu"))
+        if not op_name in _python_ops:
```

Comment:
Would there be anything wrong with:
```suggestion
        if not hasattr(sys.modules[__name__], op_name):
```
?

Reply:
Done